Formula/j/jpeg-xl.rb

```diff
@@ -63,6 +63,7 @@ def install
                     "-DCMAKE_INSTALL_RPATH=#{rpath}",
                     "-DPython_EXECUTABLE=#{Formula["asciidoc"].libexec/"bin/python"}",
                     "-DPython3_EXECUTABLE=#{Formula["asciidoc"].libexec/"bin/python3"}",
+                    "-DCMAKE_POLICY_VERSION_MINIMUM=3.5", # workaround for CMake 4
```

Comment:
This technically downgrades the 3.16 policy minimum that jpeg-xl has, which might turn some CMake policies off so watch out for that. Updating the `sjpeg` resource to the latest like upstream have is probably better.

Reply:
Ok, let's try updating it.